src/insns/malformed_shadd_clear_tag.adoc

```diff
@@ -0,0 +1,2 @@
+NOTE: This instruction sets `cd.tag=0` if `cs2` 's bounds are
```

Comment:
Should this be a common file to be included from cadd as well, just to keep all these similar instructions in sync.

Reply:
There is a small difference because in `cadd` the capability is in `cs1` rather than `cs2`. I guess we can use a variable to change the text slightly just before/after the file is included. I can give it a try